sound/soc/sof/debug.c

```diff
@@ -17,6 +17,103 @@
 #include "sof-priv.h"
 #include "ops.h"
 
+static ssize_t sof_dfsentry_write(struct file *file, const char __user *buffer,
+				  size_t count, loff_t *ppos)
+{
+	struct snd_sof_dfsentry *dfse = file->private_data;
+	struct snd_sof_dev *sdev = dfse->sdev;
+	struct sof_ipc_cmd_hdr hdr;
+	struct sof_ipc_reply reply;
+	size_t size;
+	char *string;
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_DEBUG)
```

Comment:
should we have an explicit config for IPC tests? Not sure we want this by default when the top-level DEBUG is set, do we?

Reply:
@plbossart I think it makes sense to cover it under the top-level DEBUG. If we enable top-level DEBUG, I think we should expose as may debugging features as possible. But if you disagree, I can make this specific. I dont mind either way

Reply:
@plbossart I think it makes sense to cover it under the top-level DEBUG. If we enable top-level DEBUG, I think we should expose as may debugging features as possible. But if you disagree, I can make this specific. I dont mind either way